qdrant-landing/content/documentation/cloud/authentication.md

```diff
@@ -101,3 +101,14 @@ var client = new QdrantClient(
   apiKey: "<paste-your-api-key-here>"
 );
 ```
+
+```go
+import "github.com/qdrant/go-client/qdrant"
```

Comment:
shouldn't you use `https://github.com/qdrant/go-client-v2/qdrant` in your samples directly, or isn't v2 ready to be used?

Reply:
We merge this when it's live on the main repo